app/common/locale/content.json

```diff
@@ -931,7 +929,7 @@
         "fullDescription": "Infected Blood Compensation (IBC)",
         "agency": "Infected Blood Compensation Authority",
         "agencyAcronym": "IBCA",
-        "panel": "a judge and a doctor"
+        "panel": "judge and if appropriate, a doctor and or a financial expert"
```

Comment:
I think I changed this. Are we changing it back?

Reply:
This is what Sahamat gave me for this when working on SYA. I also had the translation ready. Will confirm this with him

Reply:
Confirmed with Sahamat